extensions/Worker.Extensions.Storage.Blobs/src/Config/BlobStorageBindingOptionsSetup.cs

```diff
@@ -61,39 +61,5 @@ public void Configure(string name, BlobStorageBindingOptions options)
             options.BlobClientOptions = (BlobClientOptions)_componentFactory.CreateClientOptions(typeof(BlobClientOptions), null, connectionSection);
             options.Credential = _componentFactory.CreateTokenCredential(connectionSection);
         }
-
-        /// <summary>
-        /// Either constructs the serviceUri from the provided accountName
-        /// or retrieves the serviceUri for the specific resource (i.e. blobServiceUri or queueServiceUri)
-        /// </summary>
-        private bool TryGetServiceUri(IConfiguration configuration, out Uri serviceUri)
```

Comment:
Are we moving this logic to shared project because it is common between Blobs and Tables? We can bring this up in our feature sync that do we want to let each extension define its own authentication logic or have it in shared project. For cosmos, this is happening inside [cosmos extension](https://github.com/Azure/azure-functions-dotnet-worker/blob/feature/sdk-type-binding/extensions/Worker.Extensions.CosmosDB/src/Config/CosmosDBBindingOptionsSetup.cs). I understand this will cause duplication of logic in Blobs and Tables but then this may not be useful for remaining extensions. So, lets discuss this one.

Reply:
Yeah I moved the logic to the shared project since the code was very similar. The only difference is the subdomain being passed in, so I added that in as a parameter

Reply:
For any shared auth logic, we should have them in the shared project. If an extension is doing something different, that's fine. I don't see the downside to this